core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -194,18 +224,22 @@ public boolean isLoadBalancerCreationConfigured(Stack stack, DetailedEnvironment
     }
 
     /**
-     * Adds availability sets to instance groups associated with Knox target groups.
+     * Adds availability sets to instance groups associated with Knox and Oozie target groups.
      * This method updates the JSON parameters associated with certain instance groups.
      *
      * It's only necessary to add availability sets to the Azure deployment while we
-     * use the {@code basic} Azure Load Balancer SKU.
+     * use the {@code basic} Azure Load Balancer SKU. We use {@code standard} Load Balancer
+     * we still use availability set to indicate that we need a {@code standard} IP addresses
+     * for the instances in those availability sets.
      *
      * @param availabilitySetPrefix A string prefix. Should be a stack name normally.
      * @param loadBalancers the list of load balancers to look up target groups from.
      */
     public void configureLoadBalancerAvailabilitySets(String availabilitySetPrefix, Set<LoadBalancer> loadBalancers) {
         getKnoxInstanceGroups(loadBalancers)
                 .forEach(instanceGroup -> attachAvailabilitySetParameters(availabilitySetPrefix, instanceGroup));
+        getOozieInstanceGroups(loadBalancers)
```

Comment:
This won't clobber anything set previously for the Knox IGs, right? (Looks like both set the same thing - so essentially needs to be called on a union of all the IGs)

Reply:
Yes, this will not affect the Knox settings for the default templates. Knox and Oozie are in different instance groups. Let's say there is a custom template with Knox and Oozie in the same host group, it will override the Knox settings with the same values, so it is harmless.